### DISCUSSION TITLE
Implementing a system call for OpenBSD · poolp.org

### DISCUSSION THREAD

**Discussion by giscus** [poolpOrg/poolp.org#169] [Articles]
# Implementing a system call for OpenBSD · poolp.org

TL;DR: I found a printed copy of an assignment I had to do in 2005 back when I was a student to implement a system call for OpenBSD and Linux.

https://poolp.org/posts/2023-07-05/implementing-a-system-call-for-openbsd/

**Comment by poolpOrg** [comment]
This article was mentionned on bsdnow 530:

https://www.bsdnow.tv/530
